JDBC/OAuthBrowserFlowSampleApp/properties.example

```diff
@@ -0,0 +1,8 @@
+connectionString=jdbc:vertica://localhost:5433/VMart
+adminUser=dbadmin
+adminUserPassword=dbadmin
+oAuthUser=demouser
+authName=jdbcoauth
+clientId=clientname

```

Comment:
Some of this seems to overlap with the oauthjsonconfig.   Is this intentional?

Reply:
Yeah, I think it's a bit cleaner this way. One file is for logging in as the dbadmin and setting up the OAuth user and the other is for connecting as the OAuth user. Good to keep them separate.